benefits/static/css/styles.css

```diff
@@ -692,10 +619,10 @@ footer .footer-links li a:visited {
     display: inline-block;
   }
 
-  /* Override for only the Agency Index page */
+  /* Override for only the Agency Index page
   .agency-index .container.content h1 ~ p:nth-last-of-type(1) {
     margin-bottom: 2rem;
-  }
+  } */
```

Comment:
Should we delete this commented-out code?

Reply:
I will take care of this in future PRs - for Agency Index #950

Reply:
oops I mean #937